tests/e2e/manifests/verify_manifest_lists.sh

```diff
@@ -27,7 +27,7 @@ fi
 
 # install go if missing
 if ! `go version > /dev/null`; then
-	curl -sSL https://dl.google.com/go/go1.16.linux-amd64.tar.gz -o "/${TMP}/go.tar.gz"
```

Comment:
unrelated change @ameukam ?

Reply:
wanted to bump the version of go in the same time. Will revert.